spec/requests/oauth2_spec.rb

```diff
@@ -2,10 +2,38 @@
   let!(:login) { create(:login) }
 
   describe 'POST /token' do
+    before(:all) do
+      @orig_force_ssl = RailsApiAuth.force_ssl
+      RailsApiAuth.force_ssl = false
+    end
```

Comment:
Haha. Did you think I was just adding before/after because I was bored (and not to fix broken tests)? :)

Let me see if there is a way to test with SSL...